### DISCUSSION TITLE
Can I use this tool for 3D voxel?

### DISCUSSION THREAD

**Discussion by moshicaixi** [Project-MONAI/MONAI#3442] [Q&A]
Can I use this tool for 3D voxel segmentation? The 3D voxel is voxelized from point clouds. I think 3D voxel is similar to 3D medical image in structure, is that right? Hoping for your reply and advice.

**Comment by Nic-Ma** [comment]
Hi @moshicaixi ,

Thanks for your interest here.
@ericspod @wyli @dongyang0122 Could you please help share more info here?

Thanks in advance.

**Comment by wyli** [comment]
Yes, it should be feasible. I'm converting this to a discussion.

**Comment by dongyang0122** [comment]
It would be feasible if the voxelization is dense (not sparse).

**Reply by moshicaixi** [reply]
@dongyang0122  I guess, maybe "dense" means relatively smaller point clouds? For example, point clouds for 3D model part segmentation or indoor scene segmentation.  In my opinion, point clouds for autonomous driving is large and sparse, so the voxelization of it is also sparse. Thanks for correcting me.

**Reply by dongyang0122** [reply]
@moshicaixi are you referring a tensor type input? our current pipeline is suitable for input like 3D arrays.

**Reply by moshicaixi** [reply]
@dongyang0122 yes，I got it, thank you.
